content/zh/docs/reference/glossary/kube-apiserver.md

```diff
@@ -2,9 +2,9 @@
 title: kube-apiserver
 id: kube-apiserver
 date: 2018-04-12
-full_link: /docs/reference/generated/kube-apiserver/
+full_link: /zh/docs/concepts/overview/components/#kube-apiserver
```

Comment:
No. According to `static/_redirects`:

```
/docs/reference/generated/kube-apiserver/     /docs/reference/command-line-tools-reference/kube-apiserver/ 301
```

We should point it to the kube-apiserver reference page.

Reply:
I copy it from en page, it seems that en page need to be modified too.

Reply:
The English version works because it has the redirect ...